app/src/layouts/common/appSidebar/helpAndService/modal/requestSupportModal.jsx

```diff
@@ -75,25 +78,40 @@ const RequestSupport = ({ handleSubmit, initialize, invalid }) => {
       nextAction();
     };
   };
+  const consentHandler = (e) => {
+    setIsConsentChecked(e.target.checked);
+  };
+
+  const privocyPolicyRef = {
+    a: (
+      <LinkItem
+        link={referenceDictionary.rpEpamPolicy}
+        content={formatMessage(messages.privacyPolicy)}
+        icon={OpenIcon}
+        className={cx('inline-ref')}
+      />
+    ),
+  };
 
   return (
     <ModalLayout
-      title={formatMessage(messages.requestSupport)}
+      title={formatMessage(messages.requestService)}
       okButton={{
-        text: formatMessage(COMMON_LOCALE_KEYS.SEND),
-        onClick: (closeModal) => {
+        text: formatMessage(messages.sendRequest),
+        onClick: () => {
           handleSubmit(() => {
-            onSubmit(closeModal);
+            onSubmit(() => dispatch(hideModalAction()));
```

Comment:
Looks like this is redundant now with a new Modal layout.
The modal can be closed in the `onSubmit` function directly.